app/uk/gov/hmrc/gform/LookupLoader.scala

```diff
@@ -58,6 +58,15 @@ class LookupLoader {
   private val origin         = read("BCD-Origin.csv",        "ID",           "Name", mkAjaxLookup(ShowAll.Enabled))
   private val port           = read("BCD-Port.csv",          "PortId",       "Name", mkAjaxLookup(ShowAll.Disabled))
   private val transportMode  = read("BCD-TransportMode.csv", "ID",           "Name", mkRadioLookup)
+  private val intentBuyingWhat = read("BCD-IntentBuyingWhat.csv",        "id",           "name", mkRadioLookup)
```

Comment:
My eyes hurts, can we align all of these `val`s? Just copy paste this:

```
  private val cashType                 = read("BCD-CashType.csv",                 "ID",           "Name", mkRadioLookup)
  private val country                  = read("BCD-Country.csv",                  "CountryCode",  "Name", mkAjaxLookup(ShowAll.Disabled))
  private val currency                 = read("BCD-Currency.csv",                 "CurrencyCode", "Name", mkAjaxLookup(ShowAll.Disabled))
  private val intent                   = read("BCD-Intent.csv",                   "ID",           "Name", mkRadioLookup)
  private val intercept                = read("BCD-Intercept.csv",                "ID",           "Name", mkRadioLookup)
  private val origin                   = read("BCD-Origin.csv",                   "ID",           "Name", mkAjaxLookup(ShowAll.Enabled))
  private val port                     = read("BCD-Port.csv",                     "PortId",       "Name", mkAjaxLookup(ShowAll.Disabled))
  private val transportMode            = read("BCD-TransportMode.csv",            "ID",           "Name", mkRadioLookup)
  private val intentBuyingWhat         = read("BCD-IntentBuyingWhat.csv",         "id",           "name", mkRadioLookup)
  private val intentBigPurchase        = read("BCD-IntentBigPurchase.csv",        "id",           "name", mkRadioLookup)
  private val intentLivingCostsAndFees = read("BCD-IntentLivingCostsAndFees.csv", "id",           "name", mkRadioLookup)
  private val intentBusiness           = read("BCD-IntentBusiness.csv",           "id",           "name", mkRadioLookup)
  private val intentOther              = read("BCD-IntentOther.csv",              "id",           "name", mkRadioLookup)
  private val originWho                = read("BCD-OriginWho.csv",                "id",           "name", mkRadioLookup)
  private val originSellingSomething   = read("BCD-OriginSellingSomething.csv",   "id",           "name", mkRadioLookup)
  private val originMainPart           = read("BCD-OriginMainPart.csv",           "id",           "name", mkRadioLookup)
  private val originSavingsEarnings    = read("BCD-OriginSavingsEarnings.csv",    "id",           "name", mkRadioLookup)
```

Reply:
Done